### PR TITLE
Adds revision_oriented.revision.page.suggested.properties

### PR DESCRIPTION
**Pull request by halfak**
```
$ python
Python 3.5.1+ (default, Mar 30 2016, 22:46:26) 
[GCC 5.3.1 20160330] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mwapi
>>> from revscoring.extractors.api import Extractor
>>> extractor = Extractor(mwapi.Session("https://wikidata.org"))
>>> from revscoring.datasources import revision_oriented as ro
>>> extractor.extract(766416985, ro.revision.page.suggested.properties)
[{'rating': '0.5553102294603983', 'id': 'P31', 'url': '//www.wikidata.org/wiki/Property:P31', 'label': 'instance of', 'description': 'that class of which this subject is a particular example and member (subject typically an individual member with a proper name label); different from P279; using this property as a qualifier is deprecated—use P2868 or P3831 instead'}, {'rating': '0.5166941781838735', 'id': 'P21', 'url': ....}]
```

### CODE REVIEW

**Comment by halfak** [comment]
@Ladsgroup, can you review this?

@mariushoch, do you have any questions or concerns?

File: revscoring/extractors/api/extractor.py [hunk 1]
```diff
@@ -307,6 +310,25 @@ def get_page_creation_doc(self, page_id,
...
             return None
 
+    def get_property_suggestion_doc(self, entity_id):
+        if entity_id is None:
+            return None
+
+        logger.debug("Requesting property suggestions for ({0}) from the API"
+                     .format(entity_id))
+        doc = self.session.get(
+            action='wbsgetsuggestions', entity='Q15430523', include='all',
```
Comment (mariushoch) [inline revscoring/extractors/api/extractor.py]: Should use `entity_id`

File: revscoring/datasources/revision_oriented.py [hunk 2]
```diff
@@ -187,13 +191,32 @@ def __init__(self, name, include_creation=False):
...
             """
             :class:`~revscoring.datasources.revision_oriented.Revision` : The
             first revision to the page.
             """
 
+        if include_suggested:
+            self.suggested = Suggested(name + ".suggestions")
+            """
+            :class:`~revscoring.datasources.revision_oriented.Suggested" :
+            The set of of suggestions for a page.
```
Comment (mariushoch) [inline revscoring/datasources/revision_oriented.py]: of of

**Comment by halfak** [comment]
@mariushoch, thanks for taking a look.  I've fixed those issues.

File: revscoring/datasources/revision_oriented.py [hunk 3]
```diff
@@ -48,6 +48,7 @@ def __init__(self, name,
                  include_user_last_revision=False,
                  include_page=True,
                  include_page_creation=False,
+                 include_page_suggested=False,
```
Comment (adamwight) [inline revscoring/datasources/revision_oriented.py]: Style thing, the argument order is different here and below.  Harmless because using keyword params.

**Pull request merged**